tests/integration/cypress.json

```diff
@@ -1,3 +1,10 @@
 {
-  "baseUrl": "http://localhost:5000"
+    "baseUrl": "http://localhost:5000",
```

Comment:
Create a custom cypress.json for CI and copy it in to the Cypress image (or put it in .circleci/) rather than replacing the tests/integration/cypress.json, which should be runnable locally by default and print to stdout.

Reply:
Even with the reporter options, Cypress will show the non-xml output, but I'll move the changes to a different file.